io.github.qalculate.Qalculate.yaml

```diff
@@ -0,0 +1,107 @@
+rename-appdata-file: qalculate-gtk.appdata.xml
+rename-desktop-file: qalculate-gtk.desktop
+rename-icon: qalculate
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Network access
+  # for Update Exchange Rates
+  - --share=network
+  # Filesystem access
+  # Import CSV File
+  # Export CSV File
+  # Save Result Image
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
+  # Session Bus access
+  - --own-name=io.github.Qalculate
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+  - name: mpfr
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/mpfr/mpfr-4.0.2.tar.xz
+        sha256: 1d3be708604eae0e42d578ba93b390c2a145f17743a744d8f3f8c2ad5855a38a
+  - name: wxwidgets
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --disable-aui
+      - --disable-gstreamer8
+      - --disable-html
+      - --disable-htmphelp
+      - --disable-mdi
+      - --disable-mediactrl
+      - --disable-propgrid
+      - --disable-ribbon
+      - --disable-richtext
+      - --disable-stc
+      - --disable-webkit
+      - --disable-webview
+      - --disable-xrc
+      - --enable-unicode
+      - --with-gtk=3
+      - --without-opengl
+      - --with-libjpeg
+      - --with-libpng
+      - --with-zlib
+    cleanup:
+      - /bin
+      - /lib/wx
+      - /share/bakefile
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2
+        sha256: 440f6e73cf5afb2cbf9af10cec8da6cdd3d3998d527598a53db87099524ac807
+  - name: gnuplot
+    config-opts:
+      - --without-gd
+      - --without-libcerf
+      - --without-lua
+      - --with-readline=gnu
+      - --without-linux-vga
+      - --without-qt
+      - --enable-wxwidgets
+      - --enable-maintainer-mode
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/gnuplot/gnuplot-5.2.8.tar.gz
+        sha256: 60a6764ccf404a1668c140f11cc1f699290ab70daa1151bb58fed6139a28ac37
+  - name: libqalculate
+    config-opts:
+      - --disable-static
+      - --enable-maintainer-mode
+    sources:
+      - type: archive
+        url: https://github.com/Qalculate/libqalculate/releases/download/v3.11.0/libqalculate-3.11.0.tar.gz
+        sha256: 6992ff25774a15d6a0b09542b1f137dd5dc00995dfe9a765205ec2da39b4e13f
+  - name: qalculate-gtk
+    config-opts:
+      - --disable-static
+      - --enable-maintainer-mode
+      - --with-webkitgtk
+      - --disable-gnome-search
+    sources:
+      - type: archive
+        url: https://github.com/Qalculate/qalculate-gtk/releases/download/v3.11.0/qalculate-gtk-3.11.0.tar.gz
+        sha256: eb2a0502e9c5e93b43d2b2c42608bbafd475d70caf236a51285fea606d9d3167
+      - type: patch
+        path: qalculate-gtk-flatpak.patch
+cleanup:
```

Comment:
We tend to put the cleanup before modules and just after finish-args.

Reply:
Fixed.